### DISCUSSION TITLE
SSL error

### DISCUSSION THREAD

**Discussion by rosemarybeck5** [vercel/community#3913] [Bug Report]
### Page to Investigate

SSL error

### Steps to Reproduce

Your connection is not private
Attackers might be trying to steal your information from www.spyfans.co (for example, passwords, messages, or credit cards). Learn more
NET::ERR_CERT_AUTHORITY_INVALID
To get Chrome’s highest level of security, turn on enhanced protection

**Comment by swarnava** [comment]
Hi,

Are you still getting this error? Can you send a screenshot?

**Comment by rosemarybeck5** [comment]
It was a block from the router company , thanks !
